doc/sphinx/source/ops/index.rst

```diff
@@ -192,4 +194,4 @@ that must be performed are:
 - Transformer handlers:
 
   * Interpreter (reference) implementation of behavior. The
-    implementation should favor clarity over efficiency.
\ No newline at end of file
+    implementation should favor clarity over efficiency.
```

Comment:
What changed on this line?

Reply:
Not sure what was changed on this line. Maybe end spaces were deleted during the formatting?